components/illustratedmessage/index.css

```diff
@@ -34,19 +34,22 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-IllustratedMessage-heading {
+  color: var(--spectrum-illustratedmessage-heading-color);
+  font-size: var(--spectrum-illustratedmessage-heading-font-size);
+  font-weight: var(--spectrum-illustratedmessage-heading-font-weight);
   max-inline-size: var(--spectrum-illustrated-message-heading-max-width);
   margin: var(--spectrum-illustrated-message-heading-margin);
```

Comment:
Interesting that some of these are `--spectrum-illustrated-message` and some are `--spectrum-illustratedmessage` -- are we sure no tokens are missing?

Reply:
This is a good call-out. The `spectrum-illustrated-message` ones are defined locally, at the top of that `index.css` file, while the ones from DSS come through as `spectrum-illustratedmessage`. Perhaps we should update the locally defined ones to match the DSS output.

Thoughts?

Reply:
It's fine either way honestly, I just wanted to make sure things are building without error. Maybe go ahead and rename them for consistency, then let's get this merged!